components/hacbs/jvm-build-service/base/kustomization.yaml

```diff
@@ -11,21 +12,33 @@ namespace: jvm-build-service
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+# Skip applying the jvmbuildservice operands while the jvm-build-service operator is being installed.
+# See more information about this option, here:
+# https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#skip-dry-run-for-new-custom-resources-types
+commonAnnotations:
+  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+
+patches:
+- target:
+    kind: APIExport
+    name: jvmbuildservice
+  patch: |-
+    - op: replace
+      path: /metadata/name
+      value: jvm-build-service
+# Add APIExport flag to the operator arguments list
+- target:
+    kind: Deployment
+    name: hacbs-jvm-operator
+  patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/args/-
+      value: --api-export-name=jvm-build-service
```

Comment:
https://github.com/redhat-appstudio/jvm-build-service/pull/282 should remove the need for this patch